dll/win32/shell32/dialogs/drive.cpp

```diff
@@ -81,6 +81,27 @@ GetDefaultClusterSize(LPWSTR szFs, PDWORD pClusterSize, PULARGE_INTEGER TotalNum
         else
             return FALSE;
     }
+    else if (!wcsicmp(szFs, L"FATX"))
+    {
+        if (TotalNumberOfBytes->QuadPart <= (16 * 1024 * 1024))
```

Comment:
From what did you take these values?
Comment a URL.

Reply:
The values are taken from both FAT and FAT32 code condition path cases, if you actually pay attention to the code above.

Reply:
Ok.